game-core/src/main/java/games/strategy/engine/data/TechnologyFrontier.java

```diff
@@ -9,6 +9,9 @@
 
 import games.strategy.triplea.delegate.TechAdvance;
 
+/**
+ * A collection of {@link TechAdvance}s.
```

Comment:
This comment could be more helpful. I'm confused why a collection would have this field:

``` 
private final String m_name;
```

This is more than just a collection of tech advances; this comment is useful but a bit marginal, would be nice to get more info on what a TechnologyFrontier represents, and what role this class plays

Reply:
A `TechnologyFrontier` corresponds to the `category` element under a `playerTech` element in the XML.  It's basically the group of technologies available to a specific player.

Reply:
That response IMO would be an excellent second sentence on this javadoc.

Reply:
Already done in 4323bed20.